spyder/config/main.py

```diff
@@ -270,6 +270,14 @@
               'show_hscrollbar': True,
               'visible_if_project_open': True
               }),
+            ('vcs',
+             {
+              'use_vcs_higlighting': True,
+              'color/untracked': "#ff0000",
+              'color/ignored': "#555555",

```

Comment:
It is very difficult to read the filenames with this color in the dark theme. Can you use ``#999999`` from our Spyder Dark syntax highlighting theme palette? Thanks.